src/components/Header/index.scss

```diff
@@ -12,6 +12,7 @@
   min-width: 125px;
 }
 
+
```

Comment:
remove extra line. it's ok to have one blank line in between but we usually don't do 2